src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -13,41 +13,124 @@ tags:
 
 ---
 
-There are several restrictions around who can publish, use, and subscribe to Nerdpacks (the file packages that represent [New Relic One applications](https://developer.newrelic.com/build-apps)).
+<Intro>
 
-## Restrictions for basic users
+Understand the requirements for managing and using Nerdpacks in New Relic One.
 
-The most important permissions factor is [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type). A basic user has several restrictions related to their inability to access Full Stack Observability, and a full user theoretically has full abilities.
+</Intro>
 
-Basic users can build and serve their own Nerdpacks locally. They can also use public Nerdpacks that meet **both** of the following criteria:
+## Capabilities at a glance
 
-* New Relic must have specifically allowed basic users to use the Nerdpack
-* Someone else on the basic user's account must have already subscribed to the Nerdpack
+Your ability to manage and use Nerdpacks depends on your user's:
 
-Nerdpacks that basic users can use are rare.
+- Model
```

Comment:
Here, you might say something like, " Model (original or New Relic One model)"
I recommend this because when I saw Model, I was confused, and I'm used to this distinction. If that's too wordy, you could make it "User model" -- I know that introduces a little redundancy with the lead in.

Reply:
Yep, originally, I had "User's" at the front of all of these list items, but felt it was too verbose. I think your parenthetical suggestion works.